dataland-backend/src/main/kotlin/org/dataland/datalandbackend/utils/StorageClientUtils.kt

```diff
@@ -0,0 +1,39 @@
+package org.dataland.datalandbackend.utils
+
+import org.dataland.datalandbackend.services.StreamingExternalStorageControllerApi
+import org.dataland.datalandexternalstorage.openApiClient.api.ExternalStorageControllerApi
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import java.io.InputStream
+
+/**
+ * The class holds the various storage clients for the data manager services
+ */
+
+@Component("StorageClientUtils")
+class StorageClientUtils(
```

Comment:
ExternalStorageClientUtils wäre besser, damit man direkt sieht, mit welchem storage das spricht.

Reply:
Lass uns vermeiden, Spring-Services als "Utils" zu bezeichnen.
Util-Klassen tragen ja normalerweise einfach ausgelagerte Hilfs-funktionen, um die nicht immer wieder überall zu schreiben. Es scheint mir hier jedoch ein andere Fall vorzuliegen: Und zwar dass du Funktionalität aus dem PrivateDataManager in einen eigenen Service ausgelagert hast.  (Du hast ja auch wieder eine @Component hier)
Daher sollte das einen Namen passend zu seinem Service-Dasein tragen.
Bspw. ExternalStorageDataGetter oder sowas.

Reply:
Passe ich an